app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -175,399 +173,348 @@ class AdministratorControlsActivityTest {
+        matches(withText("Ben"))
+      ).perform(click())
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.profile_delete_button)).perform(click())
+      onView(withText(R.string.profile_edit_delete_dialog_message))
+        .inRoot(isDialog())
+        .check(
+          matches(
+            isDisplayed()
+          )
+        )
     }
   }
 
   @Test
-  fun testAdministratorControlsFragment_clickAppVersion_opensAppVersionActivity() {
+  @Config(qualifiers = "sw600dp")
+  fun testAdministratorControlsFragment_configChange_checkOpensDeletionDialog() {
     launch<AdministratorControlsActivity>(
       createAdministratorControlsActivityIntent(
-        profileId = internalProfileId
+        profileId = 1
       )
     ).use {
+      onView(ViewMatchers.isRoot()).perform(OrientationChangeAction.orientationLandscape())
       testCoroutineDispatchers.runCurrent()
-      scrollToPosition(position = 3)
-      onView(withId(R.id.app_version_text_view)).perform(click())
-      intended(hasComponent(AppVersionActivity::class.java.name))
+      onView(atPositionOnView(R.id.profile_list_recycler_view, 1, R.id.profile_list_name)).check(
+        matches(withText("Ben"))
+      ).perform(click())
+      onView(withId(R.id.profile_delete_button)).perform(ViewActions.scrollTo()).perform(click())
+      testCoroutineDispatchers.runCurrent()
+      onView(withText(R.string.profile_edit_delete_dialog_message))
+        .inRoot(isDialog())
+        .check(
+          matches(
+            isDisplayed()
+          )
+        )
     }
   }
 
   @Test
-  fun testAdministratorControls_selectAdminNavItem_adminControlsIsDisplayed() {
+  @Config(qualifiers = "sw600dp")
+  fun testAdministratorControlsFragment_configChange_checkDeletionDialogIsVisible() {
     launch<AdministratorControlsActivity>(
       createAdministratorControlsActivityIntent(
-        profileId = internalProfileId
+        profileId = 1
       )
     ).use {
-      it.openNavigationDrawer()
-      onView(withId(R.id.administrator_controls_linear_layout)).perform(nestedScrollTo())
-        .perform(click())
-      onView(withText(context.getString(R.string.administrator_controls_edit_account)))
-        .check(matches(isDisplayed()))
+      testCoroutineDispatchers.runCurrent()
+      onView(atPositionOnView(R.id.profile_list_recycler_view, 1, R.id.profile_list_name)).check(
+        matches(withText("Ben"))
+      ).perform(click())
+      onView(withId(R.id.profile_delete_button)).perform(ViewActions.scrollTo()).perform(click())
+      testCoroutineDispatchers.runCurrent()
+      onView(ViewMatchers.isRoot()).perform(OrientationChangeAction.orientationLandscape())
+      testCoroutineDispatchers.runCurrent()
+      onView(withText(R.string.profile_edit_delete_dialog_message))
+        .inRoot(isDialog())
+        .check(
+          matches(
+            isCompletelyDisplayed()
+          )
+        )
     }
   }
 
+  private fun isAdminProfileVisible() {
```

Comment:
```suggestion
  private fun checkIsAdminProfileVisible() {
```

I think this is what you meant--the function should clearly describe what it's doing.

Ditto for isAdminTextVisibile.

Reply:
done